sources/odoo.conf

```diff
@@ -0,0 +1,116 @@
+[options]
+
+
+; --------------------
+; change this ASAP!!!!
+;---------------------
+admin_passwd = somesuperstrongadminpasswdYOUNEEDTOCHANGEBEFORERUNNING
+
+; set the default timezone for openerp interface
+timezone = Europe/Paris
+
+db_maxconn = 64
+
+db_name = False
+db_template = template1
+dbfilter = .*
+
+; set this to True if you have installed unaccent in your PostgreSQL
+; and want to be able to search without accents
+unaccent = True
+
+; These options are really important and depend on the type of
+; deployment you want to have so read the doc carefully and
+; set these values accordingly
+workers = 0
+max_cron_threads = 2
+limit_memory_hard = 2147483648
```

Comment:
It's a default configuration file (I didn't do any modification in it). 
If you need to change these values, just use your own odoo.conf
Sébastien Maillard --
*Sébastien Maillard* /[Technical Analyst]/
Phone: + 86 186 2103 7589 (Cell), + 86 21 6211 8017/27/37 (Office)
*Elico Shanghai* (Shenzhen/Singapore) <http://www.elico-corp.com> /*Odoo 
Gold Partner, best Odoo Partner 2014 for APAC*/
On 09/25/2015 05:40 PM, Augustin Cisterne-Kaas wrote:
>
> In sources/odoo.conf 
> <https://github.com/Elico-Corp/odoo-docker/pull/4#discussion_r40414194>:
>
> > +db_maxconn = 64
> > +
> > +db_name = False
> > +db_template = template1
> > +dbfilter = .*
> > +
> > +; set this to True if you have installed unaccent in your PostgreSQL
> > +; and want to be able to search without accents
> > +unaccent = True
> > +
> > +; These options are really important and depend on the type of
> > +; deployment you want to have so read the doc carefully and
> > +; set these values accordingly
> > +workers = 0
> > +max_cron_threads = 2
> > +limit_memory_hard = 2147483648
>
> These options have a huge impact on Odoo depending on our server 
> configuration. It would be nice if we could assign in our 
> docker-compose.yml (ENV variable maybe).
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/Elico-Corp/odoo-docker/pull/4/files#r40414194>.
>